adafruit_display_text/scrolling_label.py

```diff
@@ -26,36 +26,42 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Display_Text.git"
 
+try:
+    from typing import Union, Optional
+    from fontio import BuiltinFont
+    from adafruit_bitmap_font.bdf import BDF
+    from adafruit_bitmap_font.pcf import PCF
```

Comment:
You can change this over to use the `fontio.FontProtocol` typing protocol!

```suggestion
    from fontio import FontProtocol
```